samples/bookinfo/kube/mixer-rule-deny-serviceaccount.yaml

```diff
@@ -12,16 +11,14 @@ apiVersion: "config.istio.io/v1alpha2"
 kind: checknothing
 metadata:
   name: denyproductpagerequest
-  namespace: istio-system
 spec:
 ---
 apiVersion: "config.istio.io/v1alpha2"
 kind: rule
 metadata:
   name: denyproductpage
-  namespace: istio-system
 spec:
   match: destination.labels["app"] == "details" && source.user == "spiffe://cluster.local/ns/default/sa/bookinfo-productpage"
   actions:
-  - handler: denyproductpagehandler.denier.istio-system
-    instances: [ denyproductpagerequest.checknothing.istio-system ]
+  - handler: denyproductpagehandler.denier.default
+    instances: [ denyproductpagerequest.checknothing.default ]
```

Comment:
ditto

Reply:
Thank you for suggesting that, @jmuk , that made user's experience easier!  I tried it and worked too!